reana-cwl.yaml

```diff
@@ -0,0 +1,28 @@
+version: 0.1.0
+metadata:
+  authors:
+   - Harri Hirvonsalo <hjhsalo@gmail.com>
+   - Anton Khodak <anton.khodak@ukr.net>
+   - Diego Rodriguez <diego.rodriguez@cern.ch>
+   - Tibor Simko <tibor.simko@cern.ch>
+  title: Hello world - A simple reusable analysis example
+  date: 13 February 2017
+  repository: https://github.com/reanahub/reana-demo-helloworld/
+code:
+  files:
+   - code/helloworld.py
+inputs:
+  files:
+    - inputs/names.txt
+  parameters:
+    input: workflow/cwl/helloworld-job.yml
+outputs:
+  files:
+   - outputs/greetings.txt
+environments:
+  - type: docker
+    image: python:2.7
+workflow:
+  type: cwl
+  file: workflow/cwl/helloworld.cwl
```

Comment:
Can please you also amend `README.rst` as follows:

- edit section "4. Analysis workflow" to add a separate Yadage and CWL sections, as we are demonstrating the use of the two workflow specifications.  You can delete the detailed yadage yaml and keep only links to Yadage and CWL specs.

- add new section "Local testing with CWL" right after "Local testing with Yadage" with similar content.

- edit section "Run example on REANA cloud" to add one sentence going like "The following example uses Yadage workflow engine. If you would like to use CWL workflow engine, please just use `-f reana-cwl.yaml`   with `reana-client` commands.

We can keep the README short, so there is not necessary to add much details... these could be added in separate CWL-oriented and Yadage-oriented docs if we need them.

(Hope you see what I mean; otherwise I can also amend the README.rst after merging...)